client/src/style/scss/ui.scss

```diff
@@ -254,44 +254,25 @@ $ui-margin-horizontal-large: $margin-v * 2;
     }
 }
 
-.ui-options {
-    .ui-options-list {
-        @extend .ui-input;
-        height: 100% !important;
-        input[type="checkbox"],
-        input[type="radio"] {
-            display: none;
-        }
-        label {
-            margin: 0px;
-            cursor: pointer;
-        }
-        .ui-drilldown-button {
-            float: left;
-            cursor: pointer;
-            font-size: 1.2em;
-            margin-right: 5px;
-            margin-top: 3px;
-        }
-        .ui-drilldown-subgroup {
-            display: none;
-            margin-left: 25px;
-        }
-        input[type="checkbox"] + label:before {
-            @extend .far;
-            margin-right: 5px;
-            content: fa-content($fa-var-square);
+.ui-form-field {
```

Comment:
Are these used anywhere else?  This would be a good time to move the styling to the drilldown component.

Reply:
Had already done a code search - didn't find anything so thusly named that commit: [Removed long-deprecated, unused css](019250131420f146894de7b624920d4b260a68e2). (But let me know if I missed anything!)

Same-page css Vue scoping restriction plus the two recursive Vue components (`FormDrilldownOption.vue` and `FormDrilldownList.vue`) means I need access to the higher-level component (if we go with suggested design to select ["all except first nested drilldown"](https://github.com/galaxyproject/galaxy/pull/18074/files#diff-5437dfc914b328943dc3904f1095bf166485f2ffd1a371bc15cc5505ba120648R273)) so that would put the drill-down scss in `FormElement.vue`, right? Otherwise, maybe I should try `v-deep` to break out of the Vue scoping restriction to get it into the drill-down Vue component itself... but that may lack browser compatibility.

Reply:
`v-deep` does not require browser support. It is a compile time directive for Vue to control scoping.

Reply:
@ElectronicBlueberry that is super-helpful, I probably had read some older information, so your reply is reassuring.